pkg/recipes/types_test.go

```diff
@@ -19,13 +19,15 @@ package recipes
 import (
 	"testing"
 
+	rpv1 "github.com/radius-project/radius/pkg/rp/v1"
 	"github.com/stretchr/testify/require"
 )
 
 func TestRecipeOutput_PrepareRecipeResponse(t *testing.T) {
 	tests := []struct {
 		desc        string
 		result      map[string]any
+		recipe      rpv1.RecipeStatus
```

Comment:
Should we add some tests with a Recipe?

Reply:
No changes to PrepareRecipeResponse as it is called only sometimes. The changes are outside.